src/realm/realmReducer.js

```diff
@@ -24,6 +24,7 @@ const initialState = {
   twentyFourHourTime: false,
   canCreateStreams: true,
   isAdmin: false,
+  isTopicMandatory: false,
```

Comment:
Let's add a `dropCache` migration in src/boot/store.js for this. Something like,

```js
  // Add `isTopicMandatory` to `state.realm`.
  '27': dropCache,
```

Reply:
That's new, I understand what this does, but when should I add a `dropCache`?

Reply:
And i think you mean `'28': dropCache` ? there is already 
`'27': state => ({
    ...state,
    accounts: state.accounts.filter(a => a.email !== ''),
  }),`